pkg/exposer/image.go

```diff
@@ -39,11 +45,24 @@ func getInheritedPodInfo(ctx context.Context, client kubernetes.Interface, veler
 	}
 
 	if len(podSpec.Containers) != 1 {
-		return podInfo, errors.Wrap(err, "unexpected pod template from node-agent")
+		return podInfo, errors.New("unexpected pod template from node-agent")
 	}
 
 	podInfo.image = podSpec.Containers[0].Image
 	podInfo.serviceAccount = podSpec.ServiceAccountName
 
+	podInfo.env = podSpec.Containers[0].Env
+	podInfo.volumeMounts = podSpec.Containers[0].VolumeMounts
+	podInfo.volumes = podSpec.Volumes
+
+	command := podSpec.Containers[0].Command
+	for i, flag := range command {
```

Comment:
May need to consider how to get the `log-format` and `log-level` parameters from the `velero node-agent server` CLI.
The current node-agent CLI looks like the following. The parameters are in the `args`, not the `command`.
Another thing is the parameter format. It could be `--log-format xxx` or `--log-format=xxx`. We cannot use the same way to read the parameter value from those two formats.

``` yaml
    spec:
      containers:
      - args:
        - node-agent
        - server
        - --features=EnableCSI
        command:
        - /velero
```

Reply:
Done, changed to get them from `Args` and cover both single-line and double-line case.